commcare_connect/opportunity/views.py

```diff
@@ -889,6 +890,10 @@ def suspend_user(request, org_slug=None, opp_id=None, pk=None):
     access.suspension_date = now()
     access.suspension_reason = request.POST.get("reason", "")
     access.save()
+
+    # Clear the cached opportunity access for the suspended user
+    remove_opportunity_access_cache(access.user, access.opportunity)
```

Comment:
You can call `_get_opp_access.clear` directly from within `access.save`

Reply:
@sravfeyn  Changed